patches/minecraft/net/minecraft/world/gen/layer/MixRiverLayer.java.patch

```diff
@@ -0,0 +1,11 @@
+--- a/net/minecraft/world/gen/layer/MixRiverLayer.java
++++ b/net/minecraft/world/gen/layer/MixRiverLayer.java
+@@ -22,6 +22,8 @@
+       if (LayerUtil.func_202827_a(i)) {
+          return i;
+       } else if (j == field_202725_h) {
++    	 net.minecraft.world.biome.Biome moddedRiverBiome = net.minecraftforge.common.BiomeManager.getRiver(Registry.field_212624_m.func_148745_a(i));
++         if (moddedRiverBiome != null) return Registry.field_212624_m.func_148757_b(moddedRiverBiome);
```

Comment:
Indentation is wrong here. There's a tab in there somewhere.

Reply:
fixed